package.json

```diff
@@ -16,9 +17,9 @@
     "url": "git@github.com:temando/gitlab-ci-variables-cli.git"
   },
   "bin": {
-    "setAllVars": "bin/setAllVariables.js"
+    "glci": "bin/glci.js"
```

Comment:
👍 

To retain BC, could we leave the old `setAllVars` as well?

Or, if we don't feel like that's important, the tool could just go to v2?

Reply:
I suggest going v2

Reply:
I'm fine with that 👍